app/Http/Controllers/Admin/DigitalPublicationArticleController.php

```diff
@@ -44,8 +45,17 @@ protected function indexData($request)
     {
         $digPub = app(DigitalPublicationRepository::class)->getById(request('digitalPublication'));
 
+        $articlesList = $this->repository->withDepth()->defaultOrder()->get()->filter(function ($article) {
+            return $article->depth < 3;
```

Comment:
If you set the maximum depth to three, where there ever be articles greater than three levels deep?

Reply:
@nikhiltri this is up in the air. Just from designs it only seems like we'll go 3 levels deep but maybe we get feedback from pubs on this